sim/midas/src/main/scala/midas/FPGAQoRShimGenerator.scala

```diff
@@ -35,41 +35,3 @@ class Midas2QoRTargets extends Config((site, here, up) => {
   }
 })
 
-
-// Generates synthesizable unit tests for key modules, such as simulation channels
-// See: src/main/cc/unittest/Makefile for the downstream RTL-simulation flow
-//
-// TODO: Make the core of this generator a trait that can be mixed into
-// FireSim's ScalaTests for more type safety
-object QoRShimGenerator extends App with midas.rocketchip.util.HasGeneratorUtilities {
-
- case class QoRShimOptions(
-      configProject: String = "midas.unittest",
-      config: String = "Midas2QoRTargets") {
-    val fullConfigClasses: Seq[String] = Seq(configProject + "." + config)
-  }
-
-  trait HasUnitTestOptions {
-    self: ExecutionOptionsManager =>
-    var qorOptions = QoRShimOptions()
-    parser.note("MIDAS Unit Test Generator Options")
-    parser.opt[String]("config-project")
-      .abbr("cp")
-      .valueName("<config-project>")
-      .foreach { d => qorOptions = qorOptions.copy(configProject = d) }
-    parser.opt[String]("config")
-      .abbr("conf")
-      .valueName("<configClassName>")
-      .foreach { cfg => qorOptions = qorOptions.copy(config = cfg) }
-  }
-
-  val exOptions = new ExecutionOptionsManager("qor")
-    with HasChiselExecutionOptions
-    with HasFirrtlOptions
-    with HasUnitTestOptions
-
-  exOptions.parse(args)
-
-  val params = getConfig(exOptions.qorOptions.fullConfigClasses).toInstance
-  Driver.execute(exOptions, () => new QoRShim()(params))
-}
```

Comment:
What's the deal with this being removed?  Is it dead code that we just didn't want to update for stage/phase?

Reply:
`ExecutionOptionsManager` is deleted from FIRRTL.
Basically, I should re-implement this trait, but I'm not so sure, the usability of this trait, so I directly deleted.